baseline/client/mind-eyes/glossary.json

```diff
@@ -333,7 +333,7 @@
     },
     "SCEPTICAL": {
```

Comment:
Does it need to change here, too, or is there some place that a skeptical -> SCEPTICAL mapping happens so that the glossary item can be displayed correctly?

Reply:
Good catch, fixed!